Library/Homebrew/test/test_language_python.rb

```diff
@@ -71,18 +73,24 @@ def test_pip_install_accepts_resource
     @venv.pip_install res
   end
 
-  def test_link_scripts_links_scripts
-    bin = (@dir/"bin")
-    dest = (@dir/"dest")
-    bin.mkpath
-    refute((bin/"kilroy").exist?)
-    refute((dest/"kilroy").exist?)
+  def test_pip_install_and_link_links_scripts
+    bin = (@dir/"bin").tap(&:mkpath)
```

Comment:
I was trying to avoid repeating `bin` again and used something every Ruby object has. I used (or maybe abused) `Object#tap`.